infrastructure/modules/aws/eks/locals.tf

```diff
@@ -18,13 +18,13 @@ resource "random_string" "random_resources" {
 locals {
   account_id                                           = data.aws_caller_identity.current.id
   region                                               = data.aws_region.current.name
-  tags                                                 = merge({ module = "eks" }, var.tags)
-  iam_worker_autoscaling_policy_name                   = "eks-worker-autoscaling-${module.eks.cluster_name}"
-  iam_worker_assume_role_agent_permissions_policy_name = "eks-worker-assume-agent-${module.eks.cluster_name}"
+  tags                                                 = merge({ module = "eks-${var.name}" }, var.tags)
+  iam_worker_autoscaling_policy_name                   = "eks-worker-autoscaling-${var.name}"
+  iam_worker_assume_role_agent_permissions_policy_name = "eks-worker-assume-agent-${var.name}"
   ima_aws_node_termination_handler_name                = "${var.name}-aws-node-termination-handler-${random_string.random_resources.result}"
   aws_node_termination_handler_asg_name                = "${var.name}-asg-termination"
   aws_node_termination_handler_spot_name               = "${var.name}-spot-termination"
-  kubeconfig_output_path                               = "${path.root}/generated/eks/kubeconfig-${var.name}"
+  kubeconfig_output_path                               = can(coalesce(var.kubeconfig_file, "")) ? var.kubeconfig_file : "${path.root}/generated/kubeconfig"
```

Comment:
not sure to understand this line.
can(coalesce(var.kubeconfig_file, "")) always true ?
is this similar to: 
kubeconfig_output_path= coalesce(var.kubeconfig_file, "${path.root}/generated/kubeconfig") ?